transform/mattermost-analytics/models/marts/sales/marketing/fct_marketing_campaign_member.sql

```diff
@@ -0,0 +1,39 @@
+with marketing as (
+    select marketing_id
+        , email as email
+        , subscribed_content
+        , server_id
+        , created_at
+        , updated_at
+    from
+        {{ ref('stg_cws__marketing')}}
```

Comment:
Shouldn't this CTE filter for rows with `subscribed_content = security_newsletter`? If other content is added in the future, the current code will mark it as "Security newsletter".

Reply:
Added the `security_newsletter` filter